### DISCUSSION TITLE
Colour slider of an Instrument

### DISCUSSION THREAD

**Discussion by kezzy1966** [jamulussoftware/jamulus#2288] [Ideas]
New feature request, more aimed at choir use, could your please consider creating a slider frame colour based on the instrument selection. So all Soprano's would appear with their frame colour as say Red, Altos Yellow, Tenors Blue, Bass Green etc. So when you have a large choir online, and the section is grouped,  that section would be easily identified. 

I know this is a big ask but I thought I would raise it. Thanks.

**Comment by corrados** [comment]
I recommend to join the following Facebook group: https://www.facebook.com/groups/jamuluschoralcommunity
They have a lot of recommendations and optimizations for using Jamulus with large choirs. If you have not already seen it, there is video of a large choir using Jamulus on Youtube: https://www.youtube.com/watch?v=KuGD7OD3emg&t=229s

You could use the "Skill" setting in "Musician Profile" to apply a specific color to a fader.

**Comment by kezzy1966** [comment]
Hi Corrados, yes I am fully aware of all those groups and read / participate in them.

The skill colour is the sort of thing I am looking for but it is needed by vocal group. I am not sure singers egos would allow themselves to be classified as 'beginner' etc to help the MD organise the group.

Also the the pastel colours are far too faint to be effective. Could they be much bolder/ primary colours so they can stand out ?

**Comment by ann0see** [comment]
@dcorson-ticino-com @mulyaj maybe you can also have a look at this? I'll unsubscribe from this issue. Please ping me if you need me.

**Comment by dcorson-ticino-com** [comment]
@ann0see I think this is already done.
There is the pastel color of the fill area beginner...etc. and a colored frame around the area depending on the instrument.
If you sort on instrument you get the colors grouped together.

**Comment by ann0see** [comment]
Moved to a discussion.
